arches_modular_reports/src/arches_modular_reports/ModularReport/ModularReport.vue

```diff
@@ -31,12 +31,14 @@ const componentLookup: ComponentLookup = {};
 
 const { graphSlug, resourceInstanceId, reportConfigName } = defineProps<{
     graphSlug: string;
-    resourceInstanceId: string;
+    resourceInstanceId: string | Ref<string>;
```

Comment:
can we just make sure that implementing apps pass in the resource instance id as a string rather than a ref?